install/uninstall.ps1

```diff
@@ -0,0 +1,125 @@
+# Enable a default -Verbose flag for debug output.
+[CmdletBinding()]
+
+# Halt script execution at the first failed command.
+$script:ErrorActionPreference='Stop'
+
+$azureauthDefaultLocation = ([System.IO.Path]::Combine($Env:LOCALAPPDATA, "Programs", "AzureAuth"))
+
+if (![string]::IsNullOrEmpty($Env:AZUREAUTH_INSTALL_DIRECTORY) `
+        -And $Env:AZUREAUTH_INSTALL_DIRECTORY -ne $azureauthDefaultLocation) {
+    Write-Warning "Ignoring AZUREAUTH_INSTALL_DIRECTORY environment variable. Custom uninstallation locations are not supported."
+}
+
+function Uninstall {
+    Close-AzureauthInstances
+
+    $locations = Get-AzureauthsInPath
+    Remove-AzureauthDirectories $locations
+    Remove-FromPath $locations

```

Comment:
We can be consistent here and rename to `Remove-AzureAuthFromPath`. Or because this is the `AzureAuth` uninstallation script, we can rename your methods to be the following:

`Kill-Process`
`Get-InPath`
`Remove-Directories`
`Remove-FromPath`

I think it will be less overloaded.

Reply:
Changed the naming